packages/hash/integration/src/tests/model/knowledge/block.model.test.ts

```diff
@@ -43,8 +41,7 @@ describe("Block model class", () => {
 
   beforeAll(async () => {
     await TypeSystemInitializer.initialize();
-    await ensureSystemTypesExist({ graphApi, logger });
-    await ensureSystemEntitiesExists({ graphApi, logger });
+    await ensureHashAppIsInitialized({ graphApi, logger });
```

Comment:
Previously in the review I asked for us to unify the initialize calls, can we just reuse this? :)

Reply:
You've done that now, thanks 🎉

Reply:
Yeah, that's what I've done in https://github.com/hashintel/hash/commit/2f497441d396eeb58da8e3e6740225ad1fd5fbf6 :)